apps/liteprotocoltester/tester_config.nim

```diff
@@ -0,0 +1,146 @@
+import
+  std/[strutils, strformat],
+  stew/results,
+  chronos,
+  regex,
+  confutils,
+  confutils/defs,
+  confutils/std/net,
+  confutils/toml/defs as confTomlDefs,
+  confutils/toml/std/net as confTomlNet,
+  libp2p/crypto/crypto,
+  libp2p/crypto/secp,
+  libp2p/multiaddress,
+  nimcrypto/utils,
+  secp256k1,
+  json
+import
+  ../../waku/common/confutils/envvar/defs as confEnvvarDefs,
+  ../../waku/common/confutils/envvar/std/net as confEnvvarNet,
+  ../../waku/common/logging,
+  ../../waku/factory/external_config,
+  ../../waku/waku_core
+
+export confTomlDefs, confTomlNet, confEnvvarDefs, confEnvvarNet
+
+const
+  LitePubsubTopic* = PubsubTopic("/waku/2/default-waku/proto")
```

Comment:
Isn't named sharding going to be deprecated? isn't it better to use static sharding?